easybuild/easyblocks/o/openfoam.py

```diff
@@ -209,19 +225,26 @@ def configure_step(self):
         if LooseVersion(self.version) < LooseVersion("2") or openfoam_extend_v3:
             self.log.debug("List of deps: %s" % self.cfg.dependencies())
             for dep in self.cfg.dependencies():
-                self.cfg.update('prebuildopts', "%s_SYSTEM=1" % dep['name'].upper())
-                self.cfg.update('prebuildopts', "{0}_LIB_DIR=$EBROOT{0}/lib".format(dep['name'].upper()))
-                self.cfg.update('prebuildopts', "{0}_INCLUDE_DIR=$EBROOT{0}/include".format(dep['name'].upper()))
+                dep_name = dep['name'].upper(),
+                dep_root = get_software_root(dep['name'])
+                dep_vars = [
+                    ("%s_SYSTEM", "1"),
```

Comment:
I knew I had a reason for using dicts. This currently doesn't work due to this line. I don't see what's so weird about not using every key in every string formatting. It's what dicts are for?